server/mailet/mock-smtp-server/README.adoc

```diff
@@ -0,0 +1,25 @@
+James Mock SMTP server
+====================
+
+This is our custom made Mock SMTP server that we use for some of our tests in James.
+
+== Dockerized Mock SMTP server
+
+Compile and build the image into a tarball with maven:
+
+    $ mvn package
+
+*Note* : You can add the option '-DskipTests' if you want to skip tests
+
+After Jib created the image, you can load it into Docker with:
+
+    $ docker load --input target/jib-image.tar
```

Comment:
giving it a name would be cool

Reply:
Jib doesn't allow renaming of a tar https://github.com/GoogleContainerTools/jib/issues/1282

But I will use a maven plugin then as suggested